.github/workflows/docs.yml

```diff
@@ -18,14 +18,16 @@ jobs:
       GIT_COMMITTER_EMAIL: ci-bot@example.com
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # for git describe
       - uses: Swatinem/rust-cache@v2
       - uses: astral-sh/setup-uv@v3
         with:
           enable-cache: true
       - name: Install Python
         run: uv python install # we use uv instead of setup-python so we get python-version resolution between our two packages
       - name: Sync
-        run: uv sync
+        run: uv sync && uv sync --no-dev --inexact --directory python
```

Comment:
They've got an issue for syncing workspaces: https://github.com/astral-sh/uv/issues/5727

It'd be swell if it did work, because I really like this pattern generally (have a shared lockfile, split up deps into the package where you need 'em).